src/Console/Command/AddPrefixCommand.php

```diff
@@ -37,19 +38,23 @@
     const OUTPUT_DIR_OPT = 'output-dir';
     /** @internal */
     const FORCE_OPT = 'force';
+    /** @internal */
+    const REPLACE_STRINGS_OPT = 'replace-strings';
 
     private $fileSystem;
     private $handle;
+    private $stringReplacer;
 
     /**
      * @inheritdoc
      */
-    public function __construct(Filesystem $fileSystem, HandleAddPrefix $handle)
+    public function __construct(Filesystem $fileSystem, HandleAddPrefix $handle, StringReplacer $stringReplacer = null)
```

Comment:
I'm not a big fan of this pattern, it's really not clear that we are changing the state of `Scoper` which is used by the handler that way, besides making the scoper stateful. What about passing the `replaceMap` as a parameter or the handler and scoper?